os/drivers/timer.c

```diff
@@ -377,6 +377,11 @@ static int timer_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	 }
 	 break;
 
+	case TCIOC_GETTIMETICK: {
+		ret = clock_systimer();
```

Comment:
ret is defined as int type, but clock_systimer() will fetch g_system_timer which is a u64 defined variable, will it be enough to just assigned over and return it?

Reply:
Please use clock() in user side.

Reply:
done